src/views/Applications/ApplicationForm.vue

```diff
@@ -486,6 +486,43 @@ export default defineComponent({
   position: relative;
   height: 36px;
   top: 4px;
-  margin-left: 16px;
+  margin-left: $kui-space-60;
+}
+
+.application-form-wrapper {
+  width: 50%;
+  margin-left:auto;
+  margin-right: auto;
+
+  .application-form-inputs,
+  .dcr-form-wrapper,
+  .reference-id-wrapper,
+  .description-wrapper,
+  .application-form-error,
+  .application-form-title {
+    margin-bottom: $kui-space-80;
+  }
+  .required-field {
+    font-size: 0.875rem;
+    line-height: 1.25rem;
```

Comment:
can we change this to use the design tokens, or the closest values at least? should probably move away from using rem.